mne/dipole.py

```diff
@@ -268,6 +268,27 @@ def plot_locations(self, trans, subject, subjects_dir=None,
             show_all, ax, block, show, scale=scale, color=color, fig=fig,
             title=title)
 
+    def to_mni(self, subject, mri_head_t, subjects_dir=None,
```

Comment:
```suggestion
    @verbose
    def to_mni(self, subject, mri_head_t, subjects_dir=None,
```

Reply:
You need the `@verbose` decorator if you have a parameter in your function signature.